src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = Lottery.getRandomBall();
+            System.out.println(ball);
+        }
     }
 }
+
```

Comment:
```suggestion
```
ideally, each file should end with exactly one empty line (not shown on github). so if there's one here, it means there are two empty lines in the file